.gitignore

```diff
@@ -133,4 +133,5 @@ dmypy.json
 .vscode
 .DS_store
 ~$*
-agage-gdrive.json
\ No newline at end of file
+agage-gdrive.json
+ben_*
```

Comment:
This shouldn't be in .gitignore for everyone. Better to just not add those files locally.